assets/aml-benchmark/components/src/aml_benchmark/benchmark_result_aggregator/main.py

```diff
@@ -14,12 +14,28 @@
     get_compute_information,
     get_parent_run_id,
     get_step_name,
-    get_mlflow_model_name_version
+    get_mlflow_model_name_version,
 )
 
 
 logger = get_logger(__name__)
 
+dataset_config_map = {
```

Comment:
what is the significance of capturing this detail?

Reply:
This is the configuration used to identify the name for the mmlu specific dataset.

Reply:
Hi @babu-namburi,

would it be possible to move this to `utils/constants.py` and rename it to `DATASET_CONFIG_2_NAME_MAP`?

Another issue will be with `truthfulqa_mc1` where model is asked to choose from multiple choices vs `truthfulqa_generation` where model is asked to generate complete answer. They both have `truthfulqa` as dataset_name but we might want to differentiate between them.